src/helm/benchmark/run_specs/melt_run_specs.py

```diff
@@ -291,3 +302,217 @@ def get_math_spec(
         + get_generative_harms_metric_specs(),
         groups=["melt"] + groups,
     )
+
+
+@run_spec_function("melt_knowledge_zalo")
+def get_melt_knowledge_zalo_spec(fewshot: bool = False) -> RunSpec:
```

Comment:
Remove the `fewshot` parameter and just use the `max_train_instances` run expander.